analyses/rna-seq-expression-summary-stats/README.md

```diff
@@ -0,0 +1,181 @@
+CDR1    ENSG00000184258.6,ENSG00000281508.1    111.1765    31.932000000000002    332.2896153846154
+```
+
+- Long table:
+
+```text
+# long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz
+CDR1    ENSG00000184258.6    Adamantimomatous craniopharyngioma    PBTA    111.1765    181.70448082054318    0.11408800038186823    Highest expressed 25%    20
+CDR1    ENSG00000281508.1    Adamantimomatous craniopharyngioma    PBTA    111.1765    181.70448082054318    0.11408800038186823    Highest expressed 25%    20
+CDR1    ENSG00000184258.6    Atypical meningioma    PBTA    31.932000000000002    50.10366423725914    0.016467672549703015    Highest expressed 25%    5
+CDR1    ENSG00000281508.1    Atypical meningioma    PBTA    31.932000000000002    50.10366423725914    0.016467672549703015    Highest expressed 25%    5
+CDR1    ENSG00000184258.6    Atypical Teratoid Rhabdoid Tumor    PBTA    332.2896153846154    1075.378786603049    0.37106139758788453    Highest expressed 25%    26
+CDR1    ENSG00000281508.1    Atypical Teratoid Rhabdoid Tumor    PBTA    332.2896153846154    1075.378786603049    0.37106139758788453    Highest expressed 25%    26
+```
+
+### Usage
+
+1. Change working directory to local `OpenPBTA-analysis`.
+2. Download data using `bash download-data.sh`. Make sure `data/gene-expression-rsem-tpm-collapsed.rds` and `data/histologies.tsv` are downloaded.
+3. Run this analysis module in the continuous integration (CI) docker image using `./scripts/run_in_ci.sh bash analyses/rna-seq-expression-summary-stats/run-rna-seq-expression-summary-stats.sh`.
+
+### Module structure
+
+```text
+.
+├── 01-tpm-summary-stats.R
+├── README.md
+├── input
+│   └── ens_symbol.tsv
+├── results
+│   ├── cancer_group_all_cohort_mean_tpm.tsv
+│   ├── cancer_group_all_cohort_mean_tpm_cancer_group_wise_quantiles.tsv
+│   ├── cancer_group_all_cohort_mean_tpm_cancer_group_wise_z_scores.tsv
+│   ├── cancer_group_all_cohort_mean_tpm_gene_wise_z_scores.tsv
+│   ├── cancer_group_all_cohort_sample_metadata.tsv
+│   ├── cancer_group_all_cohort_standard_deviation_tpm.tsv
+│   ├── cancer_group_individual_cohort_mean_tpm.tsv
+│   ├── cancer_group_individual_cohort_mean_tpm_cancer_group_wise_quantiles.tsv
+│   ├── cancer_group_individual_cohort_mean_tpm_cancer_group_wise_z_scores.tsv
+│   ├── cancer_group_individual_cohort_mean_tpm_gene_wise_z_scores.tsv
+│   ├── cancer_group_individual_cohort_sample_metadata.tsv
+│   ├── cancer_group_individual_cohort_standard_deviation_tpm.tsv
+│   ├── long_n_tpm_mean_sd_quantile_gene_wise_zscore.tsv.gz
+│   └── long_n_tpm_mean_sd_quantile_group_wise_zscore.tsv.gz
+└── run-rna-seq-expression-summary-stats.sh
+```
+
+### Analysis scripts
+
+#### 01-tpm-summary-stats.R
+
+Usage:
+
+```bash
+Rscript --vanilla '01-tpm-summary-stats.R'
+```
+
+Input:
+
+- `../../data/gene-expression-rsem-tpm-collapsed.rds`
+- `../../data/histologies.tsv`
+- `input/ens_symbol.tsv`: Gene symbol to ENSG ID conversion table. Shared by @kgaonkar6.
+
+Output:
```

Comment:
Just a note that I don't see the gzipped files when I checkout your branch - they are < 100 MB, so should be able to push them. I was able to download from this PR review, however.

Reply:
nevermind, i see them - i was looking at an earlier commit